packages/libpngwriter/CMakeLists.txt.patch

```diff
@@ -0,0 +1,32 @@
+diff -uNr pngwriter-0.7.0/CMakeLists.txt pngwriter-0.7.0.mod/CMakeLists.txt
+--- pngwriter-0.7.0/CMakeLists.txt	2018-01-22 21:10:17.000000000 +0800
++++ pngwriter-0.7.0.mod/CMakeLists.txt	2022-05-30 12:47:32.809998828 +0800
+@@ -70,11 +70,19 @@
+ 
+ # Targets #####################################################################
+ #
++
++INCLUDE_DIRECTORIES(/data/data/com.termux/files/usr/include/freetype2)
```

Comment:
Please replace `/data/data/com.termux/files/usr` with `@TERMUX_PREFIX@` in patches (which is then [automatically sed'ed](https://github.com/termux/termux-packages/blob/master/scripts/build/termux_step_patch_package.sh#L27))

Reply:
I forgot to do this :(